src/microbe_stage/editor/ToleranceInfo.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Godot;
+
+/// <summary>
+///   Displays additional tolerance slider info, such as the optimal value and value labels.
+/// </summary>
+public partial class ToleranceInfo : Control
```

Comment:
I think this class is a bit misnamed because this isn't anything to do with calculated tolerance info, or at least not directly the calculated data. I'd rather name this something like SliderOptimalMarkers or something along those lines.